### PR TITLE
chore: allow custom log formatter

### PR DESCRIPTION
**Pull request by zawan-ila**
## [PROD-4241](https://2u-internal.atlassian.net/browse/PROD-4241)

This PR adds a new settings variable to configure the way logs are formatted

### CODE REVIEW

File: course_discovery/settings/production.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
 # the values read from disk should UPDATE the pre-configured dicts.
 DICT_UPDATE_KEYS = ('JWT_AUTH',)
 
+LOGGING_FORMAT_STRING = ''
```
Comment (DawoudSheraz) [inline course_discovery/settings/production.py]: should this be empty in production.py?

Comment (DawoudSheraz) [reply course_discovery/settings/production.py]: did you mean to set this in base.py?

Comment (zawan-ila) [reply course_discovery/settings/production.py]: No. Though I have done it now. It should work either way.

File: course_discovery/settings/kubernetes.py [hunk 2]
```diff
@@ -25,8 +25,8 @@ def get_docker_logger_config():
         'disable_existing_loggers': False,
         'formatters': {
             'standard': {
-                'format': '%(asctime)s %(levelname)s %(process)d '
-                          '[%(name)s] %(filename)s:%(lineno)d - %(message)s',
+                'format': LOGGING_FORMAT_STRING or '%(asctime)s %(levelname)s %(process)d '
+                                                   '[%(name)s] %(filename)s:%(lineno)d - %(message)s'
```
Comment (DawoudSheraz) [inline course_discovery/settings/kubernetes.py:29]: nit: why is this tabbed space added?

**Review by DawoudSheraz — APPROVED** [review]

File: course_discovery/settings/kubernetes.py (hunk 2 above)
Comment (zawan-ila) [reply course_discovery/settings/kubernetes.py:29]: Just to align the lower part of the string with the top

**Pull request merged**